packages/brick-http/src/http.ts

```diff
@@ -309,9 +314,20 @@ class Http {
       } catch {
         key = config.url;
       }
+
       if (this.requestCache.has(key)) {
-        this.emit("match-api-cache", this.requestCache.size);
-        return this.requestCache.get(key);
+        const isExistRequestCacheWhite = this.requestCacheWhiteList.find(
+          (request) =>
+            request.method === config.method &&
+            (!request.url || new RegExp(request.url).test(config.url))
+        );
+        if (isExistRequestCacheWhite) {
+          this.requestCacheWhiteListKeys.add(key);
+          this.requestCache.delete(key);
```

Comment:
看了下对应的卡片，理解需求了。astrid 这里似乎并没有实现「清理缓存」的功能，应该这样？

```suggestion
          this.requestCache.clear();
```

Reply:
不是全清哦，只是对应匹配到的`key`清除哦

Reply:
@Alren-huang 我看需求是说的「清理所有当前缓存」吧？